pkg/reconciler/testing/events.go

```diff
@@ -0,0 +1,39 @@
+/*
+Copyright 2018 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package testing
+
+import (
+	"fmt"
+	"k8s.io/client-go/tools/record"
+)
+
+type EventList struct {
+	Recorder *record.FakeRecorder
+}
+
+func (l EventList) Events() []string {
+	close(l.Recorder.Events)
+	events := []string{}
+	for e := range l.Recorder.Events {
+		events = append(events, e)
+	}
+	return events
+}
+
+func Eventf(eventType, reason, messageFmt string, args ...interface{}) string {
```

Comment:
Can you also comment all of the exported types and functions from this class to satisfy the linter? The comment should start with the name of the function or type. You can see "More info" in the knative-prow-robot for more information on conventions.

Reply:
Commted as you suggested. THX.